students/11th/KimGeonWoo/main.html

```diff
@@ -0,0 +1,226 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>westargram_main_page</title>
+     <link href="style/main.css" rel="stylesheet" type="text/css" />
+  </head>
+  <body>
+    <nav class="nav_bar">
+      <div class="nav_bar_container">
+        <div class="nav_bar_logo">
+          <img class="logo_img" alt="로고 이미지" src="https://s3.ap-northeast-2.amazonaws.com/cdn.wecode.co.kr/bearu/logo.png">
+          <div class="vertical_line"></div>
+          <img class="text_logo_img" alt="텍스트 로고 이미지" src="img/logo_text.png">
+        </div>
+        <!--검색창-->
+        <div class="search_bar">
+          <input class="search_bar_input_text" type="search" placeholder="검색">
+          <div class="search_bar_container">
+            <div class="search_bar_gui">
+            <img alt="돋보기 아이콘" class="search_icon" src="img/mimi.png"></span>
+            <span>검색</span>
+          </div>
+        </div>
+      </div>
+        <!--아이콘 세개-->
+        <div class="nav_bar_icon">
+          <a href=""><img alt="탐색 아이콘" src="https://s3.ap-northeast-2.amazonaws.com/cdn.wecode.co.kr/bearu/explore.png" ></a>
```

Comment:
```suggestion
          <a href="">
            <img alt="탐색 아이콘" src="https://s3.ap-northeast-2.amazonaws.com/cdn.wecode.co.kr/bearu/explore.png" >
          </a>
```
부모 자식 요소를 잘 나타낼 수 있도록 줄바꿈 해주세요